packages/webview_flutter/webview_flutter/lib/src/webview.dart

```diff
@@ -260,6 +261,13 @@ class WebView extends StatefulWidget {
   /// By default `gestureNavigationEnabled` is false.
   final bool gestureNavigationEnabled;
 
+  /// A Boolean value that determines whether pressing a link displays a preview of the destination for the link.
+  ///
+  /// This only works on iOS.
+  ///
+  /// By default `allowsLinkPreview` is true, to match the default on iOS.
+  final bool allowsLinkPreview;
```

Comment:
Should this type be nullable, so on Platforms that doesn't support this feature, the value is null.

Reply:
I'm not a fan of nullable bools in most cases; they are annoying to use, and in practice you generally have to pick a default to treat `null` as eventually anyway.

(We shouldn't say this is `iOS` only by the way; we should instead say it's not supported on all platforms.)

Reply:
Based on the comments, assuming I should keep it as a bool and not a nullable bool. If this is wrong, let me know, and I'll change it.

Changed doc comment to 'Not supported on all platforms.'